website/content/sdk/guides/intros/stocks/stocks-screener.md

```diff
@@ -0,0 +1,314 @@
+|  1 | MSFT     |  240.33 |  -0.0059 | 17929705 |  1.77308e+12 | 25.9  |     21.49 |  1.99 |  8.73 | 10.32 | 16.53 | 39.59   |        0.198 |       0.1709 |        0.243  |        0.1301 | 0.155           |
+|  2 | GOOG     |   95.44 |  -0.0084 | 20145081 |  1.23434e+12 | 18.68 |     18.04 |  2.09 |  4.38 |  4.88 |  N/A  |  N/A    |       -0.159 |       0.121  |        0.2848 |        0.0895 |  N/A            |
+|  3 | GOOGL    |   95.19 |  -0.009  | 20047848 |  1.21605e+12 | 19.51 |     18.12 |  2.18 |  4.31 |  4.89 | 10.46 | 19.44   |        0.914 |       0.1139 |        0.321  |        0.0895 | 0.233           |
+|  4 | AMZN     |   92.42 |  -0.0163 | 65245723 |  9.3352e+11  | 85.26 |     55.08 |  3.28 |  1.86 |  6.85 | 15.91 |  N/A    |        0.549 |       0.8239 |        0.676  |        0.26   | 0.281           |
+
+Combined, the five DataFrames provide an outline of who they are, which segment of they market they belong to, and how they are currently trading.
+
+### Unconventional Applications
+
+One way to use the screener is to feed it the least amount of variables possible. In this next example, the preset file contains only one filter; requesting data for all constituents of the S&P 500 Index.
+
+```console
+# Author of preset: Danglewood
+# Description: S&P Index
+
+[General]
+Order = Market Cap.
+
+[Descriptive]
+Index = S&P 500
+
+[Fundamental]
+
+[Technical]
+```
+
+Copy and past the block above into any text editor; then save the file, as `sp500_filter.ini`, to the OpenBBUserData folder.
+
+```python
+sp500_overview = openbb.stocks.screener.screener_data(preset_loaded='sp500_filter.ini', data_type = 'overview', limit = 500)
+sp500_overview = sp500_overview.convert_dtypes()
+
+sp500_overview
+```
+
+|    | Ticker   | Company               | Sector                 | Industry                       | Country   |   Market Cap |   P/E |   Price |   Change |   Volume |
+|---:|:---------|:----------------------|:-----------------------|:-------------------------------|:----------|-------------:|------:|--------:|---------:|---------:|
+|  0 | AAPL     | Apple Inc.            | Technology             | Consumer Electronics           | USA       |  2.23252e+12 | 23.14 |  141.17 |  -0.0211 | 83578852 |
+|  1 | MSFT     | Microsoft Corporation | Technology             | Software - Infrastructure      | USA       |  1.77308e+12 | 25.9  |  240.33 |  -0.0059 | 17929705 |
+|  2 | GOOG     | Alphabet Inc.         | Communication Services | Internet Content & Information | USA       |  1.23434e+12 | 18.68 |   95.44 |  -0.0084 | 20145081 |
+|  3 | GOOGL    | Alphabet Inc.         | Communication Services | Internet Content & Information | USA       |  1.21605e+12 | 19.51 |   95.19 |  -0.009  | 20047848 |
+|  4 | AMZN     | Amazon.com, Inc.      | Consumer Cyclical      | Internet Retail                | USA       |  9.3352e+11  | 85.26 |   92.42 |  -0.0163 | 65245723 |
+...continued...
```

Comment:
Please remove this continued statements and either do `...` or put continued at the bottom. It messes up the table.

Reply:
Done